orchestra/utils/task_lifecycle.py

```diff
@@ -618,7 +576,7 @@ def get_next_task_status(task, iteration_status):
             raise IllegalTaskSubmission('Task not in rejectable state.')
 
 
-def check_worker_allowed_new_assignment(worker, task_status):
+def check_worker_allowed_new_assignment(worker):
```

Comment:
because of restaffing. the allowed task statuses were only for awaiting processing and pending review.